### ISSUE TITLE
Updating empty text locale for table is broken since 4.0.3

### ISSUE THREAD

**Issue by pmwmedia** [ant-design/ant-design#22476]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://github.com/pmwmedia/antd-empty-text-issue](https://github.com/pmwmedia/antd-empty-text-issue)

### Steps to reproduce
Run `npm start` or `npm test` in linked GitHub repo

### What is expected?
The empty text for the table should change from "Empty Init" to "Empty Loaded" as it did in the previous Ant Design versions.

### What is actually happening?
The displayed empty text doesn't update after state change.

| Environment | Info |
|---|---|
| antd | 4.0.3 |
| React | 16.13.1 |
| System | Windows 10 Pro Version 1903 |
| Browser | Chrome 80 |

---
It worked in Ant Design 4.0.0 - 4.0.2 and is broken since version 4.0.3. I can reproduce the issue on local Windows computers, but unfortunately not on codesandbox.io. Therefor, I have linked my minimal example project as GitHub repo.

**Comment by afc163** [comment]
It works on codesandbox: https://codesandbox.io/s/sparkling-voice-gypl4

Maybe you should try to remove `yarn.lock` `package-lock.json` and reinstall node_modules.

**Comment by pmwmedia** [comment]
Yes, it works on codesandbox.io. I experienced the same while creating a minimal example project. However, it doesn't work on local Windows computers. I have tested my minimal example project on multiple Windows 10 computers and laptops. Unfortunately, removing lock files doesn't help. I have already tried multiple times.

I even switched already multiple times between antd 4.0.2 and 4.0.3. Always when I run my minimal example project with antd 4.0.3 on Windows, I experience the reported issue, while running with antd 4.0.2, it works as it should without any problems. I can reproduce the different behavior of both Ant Design versions on several Windows machines.
